packages/ast/src/nodes.js

```diff
@@ -30,7 +30,7 @@ export function module(
   const node: Module = {
     type: "Module",
     id,
-    fields
```

Comment:
That file is autogenerated. We could change the generator (`scripts/generateNodeUtils.js`) to fix the style but we can also just ignore that file